src/Kunstmaan/NodeBundle/EventListener/RenderContextListener.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Kunstmaan\NodeBundle\EventListener;
+
+
+use Doctrine\ORM\EntityManager;
+use Symfony\Component\HttpKernel\Event\GetResponseForControllerResultEvent;
+use Symfony\Component\Templating\EngineInterface;
+
+/**
+ * Class RenderContextListener
+ * @package Kunstmaan\NodeBundle\EventListener
+ */
+class RenderContextListener
+{
+
+    /**
+     * @var
+     */
+    protected $em;
+
+
+    /**
+     * @var EngineInterface
+     */
+    protected $templating;
+
+    /**
+     * @param EntityManager $em
+     * @param EngineInterface $templating
+     */
+    public function __construct(EntityManager $em, EngineInterface $templating)
+    {
+        $this->em = $em;
+        $this->templating = $templating;
+    }
+
+    /**
+     * @param GetResponseForControllerResultEvent $event
+     */
+    public function onKernelView(GetResponseForControllerResultEvent $event)
+    {
+        $request    = $event->getRequest();
+        $nodeTranslation = $request->attributes->get('_nodeTranslation');
+
+        if($nodeTranslation)
```

Comment:
Check the Symfony coding standards for correct code structure (http://symfony.com/doc/current/contributing/code/standards.html - or format your code automatically in your IDE using Symfony coding standards).